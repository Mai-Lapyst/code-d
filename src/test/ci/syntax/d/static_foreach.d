static foreach (a; b) {}
void main() {
	static foreach (a; b) {}
	foreach (a; b) {}
}