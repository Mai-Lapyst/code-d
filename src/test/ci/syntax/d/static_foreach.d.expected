[{"start":0,"end":14,"scope":"keyword.control.static-foreach.d"},{"start":14,"end":15,"scope":"source.d"},{"start":15,"end":16,"scope":"source.d"},{"start":16,"end":17,"scope":"variable.d"},{"start":17,"end":18,"scope":"keyword.operator.semi-colon.d"},{"start":18,"end":19,"scope":"source.d"},{"start":19,"end":20,"scope":"variable.d"},{"start":20,"end":21,"scope":"source.d"},{"start":21,"end":25,"scope":"source.d"}]
[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":9,"scope":"entity.name.function.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":14,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":15,"scope":"keyword.control.static-foreach.d"},{"start":15,"end":16,"scope":"source.d"},{"start":16,"end":17,"scope":"source.d"},{"start":17,"end":18,"scope":"variable.d"},{"start":18,"end":19,"scope":"keyword.operator.semi-colon.d"},{"start":19,"end":20,"scope":"source.d"},{"start":20,"end":21,"scope":"variable.d"},{"start":21,"end":22,"scope":"source.d"},{"start":22,"end":26,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":8,"scope":"keyword.control.foreach.d"},{"start":8,"end":9,"scope":"source.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"variable.d"},{"start":11,"end":12,"scope":"keyword.operator.semi-colon.d"},{"start":12,"end":13,"scope":"source.d"},{"start":13,"end":14,"scope":"variable.d"},{"start":14,"end":15,"scope":"source.d"},{"start":15,"end":19,"scope":"source.d"}]
[{"start":0,"end":2,"scope":"source.d"}]